packages/styletron-react/src/dev-tool.js

```diff
@@ -55,21 +17,25 @@ type StyletronStyles = {
 };
 export const setupDevtoolsExtension = () => {
   const atomicMap = {};
-  const extensionsMap = new WeakMap();
-  const stylesMap = new WeakMap();
-  const getStyles: StyletronStyles = element => {
+  const extensionsMap = new Map();
+  const stylesMap = new Map();
```

Comment:
Any concerns about these maps growing indefinitely? The nice thing about the `WeakMaps` was if elements get garbage collected, so does the metadata.

Reply:
The same thing already happens with the `MultiCache` right? We never delete/remove unused styles. 

Since this is enabled only in the dev mode and resets with refreshing, it should be fine? If you are creating thousands of styled elements / using styletron to animate something, you would run into bigger problems even without this.

Reply:
True, although that cache is indexed on atomic styles, which I would expect to grow at a much slower rate than unique class strings. In the worst (albeit totally unrealistic) case, given `n` unique atomic styles there are `n!` possible class strings.

I don't expect this to be a problem in practice, but just something to think about.